test/testutils/locking_suite.go

```diff
@@ -1,54 +1,10 @@
 package testutils
 
 import (
-	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/solo-io/k8s-utils/testutils/clusterlock"
-	"github.com/solo-io/k8s-utils/testutils/kube"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 )
 
-// call this function with a var _ = in the test suite file
-func LockingSuiteEach() bool {
```

Comment:
I assume this is safe but can we note why this is a valid removal? I dont see a reference in the description of the pr

Reply:
Good call! updated the description